src/test/java/org/cactoos/collection/CollectionOfTest.java

```diff
@@ -79,4 +87,132 @@ public void testToStringEmpty() {
         ).affirm();
     }
 
+    @Test()
+    public void returnsIteratorWithSupportedRemove() {
+        final CollectionEnvelope<String> list = new CollectionEnvelope<String>(
+            new CollectionOf<>("eleven")
+        ) { };
+        final Iterator<String> iterator = list.iterator();
+        iterator.next();
+        iterator.remove();
+        new Assertion<>(
+            "Iterator should be empty",
+            new IterableOf<>(iterator),
+            new IsEmptyIterable<>()
+        ).affirm();
+    }
+
+    @Test
+    public void notEqualToObjectOfAnotherType() {
+        new Assertion<>(
+            "Collection is equal to object of different type",
```

Comment:
@victornoel please, change `"Collection is equal to object of different type"`  to `"Must be Collecttion equal to object of different type"`.